docker/couchdb/Dockerfile

```diff
@@ -24,9 +36,10 @@ RUN mkdir -p $WDIR
 WORKDIR ${WDIR}
 
 # get binary build of couchdb exporter
-RUN curl -ksLO https://github.com/gesellix/couchdb-prometheus-exporter/releases/download/v27.0.0/couchdb-prometheus-exporter_27.0.0_Linux_x86_64.tar.gz
-RUN tar xfz couchdb-prometheus-exporter_27.0.0_Linux_x86_64.tar.gz
-RUN rm couchdb-prometheus-exporter_27.0.0_Linux_x86_64.tar.gz LICENSE README*.md
+ARG VER=30.14.0
```

Comment:
I would suggest moving this argument variable to the top of the file and rename it to a more meaningful name (EXPORTER_VER ?).